clearblade/cloud/iot_v1/registry_types.py

```diff
@@ -0,0 +1,185 @@
+from utils import get_value
+
+class EventNotificationConfig:
+    def __init__(self, pub_sub_topic_name, subfolder_matches=None) -> None:
+        self._pub_sub_topic_name = pub_sub_topic_name
+        self._sub_folder_matches = subfolder_matches
+
+    @property
+    def pub_sub_topic_name(self):
+        return self._pub_sub_topic_name
+
+    @property
+    def sub_folder_matches(self):
+        return self._sub_folder_matches
+
+class DeviceRegistry:
+    def __init__(self, id:str = None, name:str = None,
+                 eventNotificationConfigs:list = [],
+                 stateNotificationConfig:dict = {'pubsubTopicName': ''},
+                 mqttConfig:dict = {'mqttEnabledState':'MQTT_ENABLED'},
+                 httpConfig:dict = {'httpEnabledState':'HTTP_ENABLED'},
+                 logLevel:str = None, credentials:list = []) -> None:
+        self._id = id
+        self._name = name
```

Comment:
_name is autogenerated by ClearBlade services so we don't need this. Currently, registry creation is failing because we are sending the name attribute.

Reply:
@rajasd27  i removed name from the request also there were some other errors so fixed them as well  i tested and create registry is also working fine.